### ISSUE TITLE
restrict cpu/ram usage

### ISSUE THREAD

**Issue by SpheMakh** [SpheMakh/Stimela#263]
pinging @sfebruary

**Comment by SpheMakh** [comment]
Restricting memory usage seems to be a bit tricky.  See

https://stackoverflow.com/questions/48685667/what-does-docker-mean-when-it-says-memory-limited-without-swap 

But it seems to be fixed in versions >= 17.12.1
https://github.com/moby/moby/issues/35123

**Comment by SpheMakh** [comment]
It doesn't lead to crash, just prints out a warning: 
```
WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
```

So I'll leave the fix in
